apps/admin/frontend/package.json

```diff
@@ -8,8 +8,11 @@
     "prodserver"
   ],
   "scripts": {
-    "build": "pnpm type-check && vite build",
+    "build": "pnpm --filter $npm_package_name... build:self",
+    "build:self": "pnpm type-check && vite build",
     "build:stubs": "script/build-stubs fs:src/stubs/fs.ts glob:src/stubs/glob.ts os:src/stubs/os.ts jsdom:src/stubs/jsdom.ts",
+    "clean": "pnpm --filter $npm_package_name... clean:self",
+    "clean:self": "rm -rf build",
```

Comment:
Why don't frontends get `tsc --build --clean tsconfig.build.json`?

Reply:
Because frontends don't use `tsc` to build. They use `vite build`, which internally uses `rollup` and doesn't do incremental builds across runs, so there's no build cache files to clear.